db/migrate/20190913125905_migrate_subjects.rb

```diff
@@ -0,0 +1,13 @@
+class MigrateSubjects < ActiveRecord::Migration[5.2]
+  def change
+    say_with_time 'populating/migrating course subjects' do
+      all_subject = Subject.all
+      all_course_includes_ucas_subjects = Course.includes(:ucas_subjects, :provider, :subjects)
+      all_course_includes_ucas_subjects.each do |course|
+        course.level = course.ucas_level
+        course.subjects = all_subject.where subject_name: course.dfe_subjects.map(&:to_s)
+        course.save
+      end
+    end
+  end
```

Comment:
If you want to do more testing, it might be handy to have a `down` for this too:

```
  def down
    CourseSubject.connection.truncate :course_subject
    Course.all.each { |c| c.update_column(:level, nil) }
  end
```